lms/templates/dashboard.html

```diff
@@ -11,21 +11,21 @@
 from django.utils.translation import ugettext as _
 from django.template import RequestContext
 from common.djangoapps.entitlements.models import CourseEntitlement
-from third_party_auth import pipeline
-from util.date_utils import strftime_localized
+from common.djangoapps.third_party_auth import pipeline
+from common.djangoapps.util.date_utils import strftime_localized
 from opaque_keys.edx.keys import CourseKey
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 from openedx.core.djangolib.js_utils import dump_js_escaped_json, js_escaped_string
 from openedx.core.djangolib.markup import HTML, Text
 
-from student.models import CourseEnrollment
+from common.djangoapps.student.models import CourseEnrollment
 %>
 
 <%
   cert_name_short = settings.CERT_NAME_SHORT
   cert_name_long = settings.CERT_NAME_LONG
-  catalog_link = urljoin(settings.XPRO_BASE_URL, "catalog")
+  catalog_link = urljoin(settings.MARKETING_SITE_BASE_URL, "catalog")
 %>
 
 
```

Comment:
I'm not sure why did you not face this error but I think this template needs some updates regarding `is_course_voucher_refundable`. When I set up this theme branch and try to navigate to the dashboard I see:
```
File "/tmp/mako_lms/3a27452f95b64e78143a9d7fc429fdf8/mitxpro-theme/lms/templates/dashboard.html.py", line 244, in render_body
edx.devstack.lms                  |     runtime._include_file(context, 'dashboard/_dashboard_course_listing.html', _template_uri, course_overview=course_overview, course_card_index=dashboard_index, enrollment=enrollment, is_unfulfilled_entitlement=is_unfulfilled_entitlement, is_fulfilled_entitlement=is_fulfilled_entitlement, entitlement=entitlement, entitlement_session=entitlement_session, entitlement_available_sessions=entitlement_available_sessions, entitlement_expiration_date=entitlement_expiration_date, entitlement_expired_at=entitlement_expired_at, show_courseware_link=show_courseware_link, cert_status=cert_status, can_refund_entitlement=can_refund_entitlement, can_unenroll=can_unenroll, credit_status=credit_status, show_email_settings=show_email_settings, course_mode_info=course_mode_info, is_paid_course=is_paid_course, verification_status=course_verification_status, course_requirements=course_requirements, dashboard_index=dashboard_index, share_settings=share_settings, user=user, related_programs=related_programs, display_course_modes_on_dashboard=display_course_modes_on_dashboard, show_consent_link=show_consent_link, enterprise_customer_name=enterprise_customer_name, resume_button_url=resume_button_url, partner_managed_enrollment=partner_managed_enrollment)
edx.devstack.lms                  |   File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/mako/runtime.py", line 795, in _include_file
edx.devstack.lms                  |     callable_(ctx, **kwargs)
edx.devstack.lms                  | TypeError: render_body() missing 1 required positional argument: 'is_course_voucher_refundable'
```

Trying more to test this i was able to get the theme to working successfull by:
- Adding `is_course_voucher_refundable = (session_id in enrolled_courses_voucher_refundable)` in `dashboard.html` Line#213
- Adding `is_course_voucher_refundable=is_course_voucher_refundable` in arguments in line#221

Reply:
@arslanashraf7 I tried to produce it on locally, here are the things that I did to produce it. 
- Provisioning the` master` instead of `maple` locally
- Applied this theme changes to platform with the following settings in `devstack.py`

```
ENABLE_COMPREHENSIVE_THEMING = True
COMPREHENSIVE_THEME_DIRS = [
    "/edx/src/themes/"
]
TEMPLATES[1]["DIRS"] = _make_mako_template_dirs
derive_settings(__name__)
MARKETING_SITE_BASE_URL = 'http://localhost:8053/'
```
- Setup courses in MitxPro locally.
- Enrolled in a course and can see the course on my MitxPro Dashboard.
- Clicked on the course to start it and it redirected me to platform with course contents.
- I can see themes is applied over there.
- I can see the `LMS` dashboard as well without any crash.

Is there any additional steps to do ?

Reply:
Based on the above and the discussion with @asadiqbal08 it looks like it is still not reproducible for him & eventually user is not going to land on the LMS/dashboard anyways while xPRO theme is applied. So, I think we can go ahead and merge this one.